src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Is php.ini being written from config.xml.  Maybe it's changed by last I looked think it was being written from code with heredoc string, not from the config.